src/workerd/api/crypto/crypto.c++

```diff
@@ -676,13 +679,113 @@ kj::String Crypto::randomUUID() {
 // =======================================================================================
 // Crypto Streams implementation
 
+class CRC32DigestContext final: public DigestContext {
+ public:
+  CRC32DigestContext(): value(crc32(0, Z_NULL, 0)) {}
+  virtual ~CRC32DigestContext() = default;
```

Comment:
Shouldn't this be in the parent class, not in here?

```suggestion
  ~CRC32DigestContext() = default;
```

Reply:
As I said previously, it really really doesn't matter for a pure abstract base class with no members 😅